plugins/auth-backend/src/identity/TokenFactory.ts

```diff
@@ -221,6 +218,10 @@ export class TokenFactory implements TokenIssuer {
       );
     }
 
+    // Store the user info in the database upon successful token
+    // issuance so that it can be retrieved later by limited user tokens
+    await this.userInfoDatabaseHandler.addUserInfo({ claims });
```

Comment:
The TS type excludes `'aud' | 'iat' | 'iss' | 'uip'` but they are still included here. I'd say that they are worth omitting (at least `uip`)